django/template/base.py

```diff
@@ -786,7 +788,7 @@ def render(self, context):
         bits = []
         for node in self:
             if isinstance(node, Node):
-                bit = self.render_node(node, context)
+                bit = node.render_node(context)
```

Comment:
If you're inlining `render_node`, this shoud be `rode.render(context)`, shouldn't it?